torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2426,9 +2423,59 @@ Tensor* tensorexpr::computeOperandValue(
               }
             }
 
-            return tensorOrConstant(inputs[0], indices);
+            return broadcast(c10::get<BufHandle>(inputs[0]), indices);
           });
     }
+    case aten::t: {
+      auto shape = valueShape(inputs[0]);
+      if (shape.size() == 1) {
+        return new Tensor(c10::get<BufHandle>(inputs[0]).node(), nullptr);
+      }
+      return computeOperandValue(
+          aten::transpose,
+          {inputs[0], (int64_t)1, (int64_t)0},
+          outputShape,
+          outputType);
+    }
+    case aten::transpose: {
```

Comment:
Can we add some correctness tests for these? We could write them without relying on the fuser, we can feed graphs directly to `TensorExprKernel` like it is done, e.g. here:
https://github.com/pytorch/pytorch/blob/b5b158a6c6de94dfb983b447fa33fea062358844/test/test_tensorexpr_pybind.py#L125-L134